pkg/remote/aws/s3_bucket_supplier.go

```diff
@@ -2,14 +2,15 @@ package aws
 
 import (
 	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/zclconf/go-cty/cty"
```

Comment:
Can you revert theses ordering chunk ?

Reply:
These changes are from gofmt, shouldn't we follow these rules? My IDE does that by default but I guess I can disable that